sources/packages/backend/libs/sims-db/src/entities/disbursement-schedule.model.ts

```diff
@@ -181,6 +181,9 @@ export class DisbursementSchedule extends RecordDataModel {
     (disbursementSchedule: DisbursementSchedule) =>
       disbursementSchedule.msfaaNumber,
   )
+  @Column({
```

Comment:
mm, why is this change? the name is joined in below code right, 
![image](https://github.com/bcgov/SIMS/assets/77353155/7c84fbe2-c405-4ae1-bdb9-3f119ab11310)